UI/MainScreen.cpp

```diff
@@ -1260,12 +1260,10 @@ void MainScreen::sendMessage(const char *message, const char *value) {
 		}
 		if (!strcmp(message, "browse_folderSelect")) {
 			std::string filename = value;
-			INFO_LOG(SYSTEM, "Got folder: '%s'", filename.c_str());
-			int tab = tabHolder_->GetCurrentTab();
-			// Don't allow browsing in the other tabs (I don't think it's possible to reach the option though)
-			if (tab == 1) {
-				gameBrowsers_[tab]->SetPath(Path(filename));
-			}
+			INFO_LOG(SYSTEM, "Got folder: '%s'", filename.c_str());;
+			// switch to the 'Games' tab which has the file browser
+			tabHolder_->SetCurrentTab(1);
```

Comment:
If you had no recent before, then it's actually tab 0.  I guess you could check if the first view's tag is `"MainScreenRecentGames"` and use 1 if it is.

I realize the old code hardcoded 1 so was actually wrong.

-[Unknown]